build.sbt

```diff
@@ -151,6 +151,8 @@ lazy val s3 = monixConnector("s3", Dependencies.S3).aggregate(awsAuth).dependsOn
 
 lazy val gcs = monixConnector("gcs", Dependencies.GCS)
 
+lazy val es =  monixConnector("es", Dependencies.ES)
```

Comment:
I think it would be better to rename the module name to `elasticsearch`, since it would also end up being the artifact name, and es is not very representative, agree?

Reply:
Yes